extension/build

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python3
```

Comment:
Hmm, I think that breaks the script (at least on linux/osx) because she bang is needed for the shell to run it as a python script. Can we keep it? I think you'll need to run it as `python3 extension/build` on Windows anyway, so it shouldn't hurt to keep it.

Reply:
oh yeah that's okay.